server/resolvers/TestPlanVersion/testPlanReportsResolver.js

```diff
@@ -1,28 +1,19 @@
 const {
     getTestPlanReports
-} = require('../../models/services.deprecated/TestPlanReportService');
+} = require('../../models/services/TestPlanReportService');
 
 const testPlanReportsResolver = async (
     { id: testPlanVersionId },
-    { isFinal }
+    { isFinal },
+    context
 ) => {
-    const where = {
-        testPlanVersionId
-    };
+    const { transaction } = context;
 
-    const reports = await getTestPlanReports(
-        null,
-        where,
-        null,
-        null,
-        null,
-        null,
-        null,
-        null,
-        {
-            order: [['createdAt', 'desc']]
-        }
-    );
+    const reports = await getTestPlanReports({
```

Comment:
This change is great to see! No more "counting" parameters!

Reply:
Yes!